GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg

```diff
@@ -0,0 +1,184 @@
+@PART[M2X_Tailboom|M2X_InlineIntake|M2X_AeroIntake|M2X_circularintake|M2X_EngineShroud|M2X_MantaIntake|M2X_Precooler|M2X_Shockcone]:FOR[RealismOverhaul]
+{
+	%skinMaxTemp = 2500
+}
+
+@PART[M2X_Tailboom]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.915
+}
+
```

Comment:
These intakes seem to end up with two intake modules ingame. Any stock modules and resources should be removed

Reply:
They are not, a bunch of stock modules end up still on the parts. These should be removed.